packages/search/addon/components/navi-search-bar.hbs

```diff
@@ -1 +1,15 @@
-{{yield}}
+{{! Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. }}
+<div class="navi-search" ...attributes>
+  <BasicDropdown @matchTriggerWidth={{true}} @verticalPosition="below" as |dd|>
```

Comment:
Do you want to switch this to power-select? It handles all the keyboard navigation.